config/webpack.config.production.js

```diff
@@ -56,9 +56,9 @@ module.exports = merge(config, {
       {
         test: /\.scss$/,
         include: [
-          /src\/client\/assets\/javascripts/,
-          /src\/client\/assets\/styles/,
-          /src\/client\/scripts/
+          path.resolve(__dirname, "../src/client/assets/javascripts"),
+          path.resolve(__dirname, "../src/client/assets/styles"),
+          path.resolve(__dirname, "../src/client/scripts")
```

Comment:
Can you please change quotes to be single quotes, since that's what i use in my codebase for strings?